nixos/modules/services/taler/exchange.nix

```diff
@@ -0,0 +1,184 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  this = config.services.taler.exchange;
```

Comment:
```
  cfg = config.services.taler.exchange;
```

Reply:
I think this is a matter of preference. @Atemu suggested we use `this` instead of `cfg` and I don't really mind that.

Reply:
It is absolute standard to refer to the options defined in the same file with cfg unless there is a conflict or other things which make this not a good fit. I don't think we have that here.

Reply:
I agree with @SuperSandro2000, there's no other instance of `this` being used in such a way in nixpkgs AFAICT, using `cfg` is a very established idiom.

Reply:
Alright, I'll rename it to `cfg`, then.